sdk/communication/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+To get started with any of the packages, change directory to the package you want to install and run the `pip install .` command. This will install all of the local files necessary for you to run the corresponding tests. It's important to note that if you made changes to the local files and want to run the tests again, you must run the `pip install .` command from the package root folder to update the files with your new changes.
+
+Once the package has been installed on your machine, let's jump on how to run the tests to see that everything is in order.
+
+## Testing
+
+When you go inside the tests folder of the package you are working with, you will see a folder called `recordings`. This folder contains, as its name suggests, recordings of successful calls to the API that allow us to run the tests in PLAYBACK mode and remove the necessity of hitting the actual resources every time we may want to test.
+
+### PLAYBACK
+
+To run the tests in PLAYBACK mode, set an environment variable called `AZURE_TEST_RUN_LIVE` and set its value to `false` (If the variable if not set, the default will be `false`). After your variable has been set, change directory to the `tests` folder of the package you're working on and run the `pytest .` command.
+
+If the tests are successful, we can proceed to run the tests in LIVE mode.
+
+### LIVE
+
+Because in LIVE mode we are hitting an actual resource, we must set the appropriate environment variable to make sure the code tests against the resource we want. Set up an env variable called `AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING` and set it to the connection string of the resource you want to test against. 
+
+Depending on which package you are testing, it may need special environment variables to test succesfully. The names of these variables can be found inside each test file in the `setUp()` function. Make sure to set these variables before running the tests themselves. You may need to restart your development environment after creating or updating these environment variables.
+
+You can run the `pytest .` command after setting the `AZURE_TEST_RUN_LIVE` variable to `true`.
+
+### Managed Identity Tests
+
+If you ran the tests in LIVE mode, you may have noticed that the files inside the recordings folder were updated. If any of the tests failed, you will see the error message right there in the recording file as well as in your terminal logs.
+
+The most probable thing is that the managed identity tests will fail at first. This is because we haven't set up any managed identity credentials for the DefaultAzureCredential object inside the tests to reference to. There are multiple ways of creating a managed identity credential.
+
+One of the easiest ways is to install the Azure CLI and run the `az login` command. If you are listed as a contributor of the resource you are testing against, this should be enough for the DefaultAzureCredential object to get the corresponding Azure Active Directory credentials you need.
```

Comment:
Should we include this tip? 
Might need to run this to create new resource
`az ad sp create-for-rbac --name <app-name>`

Reply:
Totally forgot about this one! Thank you

Reply:
I added a link pointing to the quickstart guide in the docs.microsoft.com site